src/RecyclableMemoryStream.cs

```diff
@@ -210,6 +231,18 @@ public RecyclableMemoryStream(RecyclableMemoryStreamManager memoryManager, Guid
         /// <param name="requestedSize">The initial requested size to prevent future allocations</param>
         /// <param name="initialLargeBuffer">An initial buffer to use. This buffer will be owned by the stream and returned to the memory manager upon Dispose.</param>
         internal RecyclableMemoryStream(RecyclableMemoryStreamManager memoryManager, Guid id, string tag, int requestedSize, byte[] initialLargeBuffer)
```

Comment:
do we keep this for backcomp reason? even it's internal access?

Reply:
Agreed! It should be removed.

Reply:
Will check on this. Might be a relic.